lms/djangoapps/rss_proxy/models.py

```diff
@@ -23,3 +23,6 @@ class Meta(object):
 
     def __unicode__(self):
         return six.text_type(self.url)
+
+    def __str__(self):
```

Comment:
Another case where the [python_2_unicode_compatible](https://docs.djangoproject.com/en/1.11/topics/python3/#str-and-unicode-methods) decorator should be used on the class instead of implementing both `__str__` and `__unicode__` separately.

Reply:
changes made as suggested